packages/building-utils/forked-babel-plugin-bare-import-rewrite.js

```diff
@@ -64,19 +64,16 @@ function absResolve(importPath, sourceFileName, pluginOptions = {}) {
     return importPath;
   }
 
-  const result = resolve.sync(importPath, {
+  return resolve.sync(importPath, {
     basedir: basedirResolve(importPath, sourceFileName, pluginOptions),
     extensions: pluginOptions.extensions || ['.mjs', '.js', 'json'],
     moduleDirectory: pluginOptions.resolveDirectories || 'node_modules',
-    preserveSymlinks: false,
+    preserveSymlinks: true,
     packageFilter(packageJson) {
       packageJson.main = packageJson.module || packageJson['jsnext:main'] || packageJson.main;
       return packageJson;
     },
   });
-
-  // use node's resolve to convert symlinks to real paths
```

Comment:
this was actually added on purpose to fix https://github.com/open-wc/open-wc/issues/459

the intention is to resolve symlinks to their actual paths so that we don't get duplicate imports

Reply:
Ok, I thought that this was added on purpose but I didn't find why. I'm going to close the PR